src/test/java/org/sonatype/repository/conan/internal/ui/ConanBrowseNodeGeneratorTest.java

```diff
@@ -45,9 +47,25 @@ public void canComputeNonPackagePath() {
     Asset asset = createAsset(
         "/v1/conans/jsonformoderncpp/2.1.1/vthiery/stable/download_urls");
 
-    List<String> assetPath = underTest.computeAssetPath(asset, component);
+    List<BrowsePaths> assetPath = underTest.computeAssetPaths(asset, component);
 
-    assertThat(assetPath, contains("vthiery", "jsonformoderncpp", "2.1.1", "stable", "download_urls"));
+    assertPaths(assetPath, asList("vthiery", "jsonformoderncpp", "2.1.1", "stable", "download_urls"));
+  }
+
+  private void assertPaths(List<BrowsePaths> paths, List<String> expectedPaths) {
+    assertThat(expectedPaths.size(), is(expectedPaths.size()));
```

Comment:
this assert doesn't seem particularly useful ;)

Reply:
Woops! Yeah, I'll get that fixed 😁

Reply:
Resolved in: db950ba